src/frontend/src/components/common/InputText/InputText.tsx

```diff
@@ -106,20 +117,36 @@ const InputText = ({
         </p>
       )}
       {!readOnly ? (
-        <input
-          id={inputId}
-          type={type}
-          value={value || ''}
-          placeholder={placeholder}
-          onChange={onChange}
-          onFocus={onFocus}
-          className="form-control"
-          required={required}
-          disabled={isDisabled}
-          readOnly={isAccessLink}
-          ref={inputRef}
-          aria-label={inputId}
-        />
+        checkTextArea() ? (
+          <textarea
+            id={inputId}
+            value={value || ''}
+            placeholder={placeholder}
+            onChange={onChange}
+            onFocus={onFocus}
+            className="form-control"
+            required={required}
+            disabled={isDisabled}
+            readOnly={isAccessLink}
+            ref={inputRef}
+            aria-label={inputId}
+          />
+        ) : (
+          <input
+            id={inputId}
+            type={type}
+            value={value || ''}
+            placeholder={placeholder}
+            onChange={onChange}
+            onFocus={onFocus}
+            className="form-control"
+            required={required}
+            disabled={isDisabled}
+            readOnly={isAccessLink}
+            ref={inputRef}
+            aria-label={inputId}
```

Comment:
If all these props are same for both input and textarea, can we create a common constant and use the variable instead of repeating everything. something like `.<input {..props} />`

Reply:
meaning? input and textarea are HTML elements, not a common component here. Unless react is smart to decipher each member and populate the HTML element appropriately.

Reply:
React indeed is smart enough take care of that. Here's what I mean:
```
         const props = {
            id: inputId,
            type,
            value: value || '',
            placeholder,
            onChange,
            onFocus,
            className:"form-control",
            required,
            disabled: isDisabled,
            readOnly: isAccessLink,
            ref: inputRef,
            aria-label: inputId,
          }

and the usage ...

           <input {...props} />
           <textarea {...props} />
```

Reply:
or at-least I think it is. We can give it a shot for readability and avoiding duplicity.